changelogs/head.asciidoc

```diff
@@ -20,20 +16,12 @@ https://github.com/elastic/apm-server/compare/7.13\...master[View commits]
 [float]
 ==== Bug fixes
 - Fix apm_error_grouping_name and apm_convert_destination_address {pull}5876[5876]
```

Comment:
@axw -- I'm still curious about this bug fix. I wonder if you missed my comment here: https://github.com/elastic/apm-server/pull/6083#issue-983312290

Reply:
Sorry, I did miss your question. Yes we should probably move that to 7.14.1.